api/src/main/java/org/openmrs/validator/ConditionValidator.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public License,
+ * v. 2.0. If a copy of the MPL was not distributed with this file, You can
+ * obtain one at http://mozilla.org/MPL/2.0/. OpenMRS is also distributed under
+ * the terms of the Healthcare Disclaimer located at http://openmrs.org/license.
+ *
+ * Copyright (C) OpenMRS Inc. OpenMRS is a registered trademark and the OpenMRS
+ * graphic logo is a trademark of OpenMRS Inc.
+ */
+package org.openmrs.validator;
+
+import org.openmrs.Condition;
+import org.openmrs.annotation.Handler;
+import org.springframework.validation.Errors;
+import org.springframework.validation.ValidationUtils;
+import org.springframework.validation.Validator;
+
+/**
+ * Validates {@link Condition} objects
+ */
+@Handler(supports = {Condition.class}, order = 50)
+public class ConditionValidator implements Validator {
+
+	@Override
+	public boolean supports(Class<?> aClass) {
+		return ConditionValidator.class.isAssignableFrom(aClass);
+	}
+
+	@Override
+	public void validate(Object object, Errors errors) {
```

Comment:
Are you in position to create some tests for this class? Take a look at these examples: https://github.com/openmrs/openmrs-core/tree/master/api/src/test/java/org/openmrs/validator

Reply:
Done